libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2279,7 +2280,13 @@ void GCS::setup_uarts(AP_SerialManager &serial_manager)
         chan(i).setup_uart(serial_manager, AP_SerialManager::SerialProtocol_MAVLink, i);
     }
 
-    frsky.init();
+    if (frsky == nullptr) {
```

Comment:
Shouldn't this be some sort of assert?

Reply:
I had the thought. It looks like we only ever call it once at the moment, but nothing strictly enforced that, and I didn't chase if `setup_uart` is actually fine a second time. But if you think it should be a sitl panic I can make it do that.